source/error_report/src/main.rs

```diff
@@ -0,0 +1,213 @@
+use chrono::{prelude::*, DateTime};
+use std::time::{Duration, Instant};
+use std::{
+    env,
+    fs::{self, File},
+    io::prelude::*,
+    io::{BufRead, BufReader},
+    path::Path,
+    process::{Command, Stdio},
+    str,
+};
+use toml::{map::Map, value::Value};
+use zip::write::FileOptions;
+
+fn main() {
+    let mut file_path = String::new();
+    #[allow(unused_assignments)]
+    let mut our_args = Vec::new();
+
+    let args: Vec<String> = env::args().collect();
+    if args.len() > 1 {
+        for argument in &args {
+            if argument.ends_with(".rs") {
+                file_path = argument.clone();
+            }
+        }
+        our_args = args[2..].to_vec();
+    } else {
+        println!("Usage: error_report <file_name>");
+        return;
+    }
+
+    let program_dir = args[1].clone();
```

Comment:
I assume this is guaranteed to be passed in via the implementation of the `--error-report` flag, but that is still an assumption being made, so prob should be documented as a comment

Reply:
On further thought, maybe this should be passed in as an environment variable? For example `VERUS_HOME` or such?

Reply:
Fair, I think it is fine to leave it as an argument here